ads/google/imaVideo.js

```diff
@@ -428,9 +432,15 @@ export function imaVideo(global, data) {
     // Create our own tap listener that ignores tap and drag.
     bigPlayDiv.addEventListener(mouseMoveEvent, onBigPlayTouchMove);
     bigPlayDiv.addEventListener(mouseUpEvent, onBigPlayTouchEnd);
-    bigPlayDiv.addEventListener('tapwithoutdrag', onClick.bind(null, global));
+    bigPlayDiv.addEventListener(
+        'tapwithoutdrag',
+        onBigPlayClick.bind(null, global)
```

Comment:
Use arrow funcs here instead of bind.

Might wanna consider back-changing the other events but it's fine to leave as-is.

Reply:
Personally, I think it'd be more beneficial to have it stay consistent. Helps maintainability in my opinion for someone jumping in, rather than trying to understand why this one is different. 

I definitely agree they should all be arrow functions, but I think that should be done in a larger refactor.

What do you think? 😄

Reply:
sg.